frontend/taipy/src/ScenarioDag.tsx

```diff
@@ -133,14 +142,23 @@ const ScenarioDag = (props: ScenarioDagProps) => {
         if (displayModel) {
             setScenarioId(displayModel[0]);
             // populate model
-            doLayout = populateModel(displayModel, model);
+            doLayout = populateModel(addStatusToDisplayModel(displayModel, taskStatuses), model);
+        }
+        const rects = engine.getModel() && engine
+            .getModel()
+            .getNodes()
+            .reduce((pv, nm) => {
```

Comment:
No issue with initial value for pv?
If this works, I guess not.

Reply:
No